coreneuron/apps/main1.cpp

```diff
@@ -32,33 +32,33 @@ THE POSSIBILITY OF SUCH DAMAGE.
  * @brief File containing main driver routine for CoreNeuron
  */
 
+#include <cstring>
+#include <climits>
 #include <vector>
-#include <string.h>
+
 #include "coreneuron/engine.h"
```

Comment:
Maybe those includes could be arranged better? Like sorting them alphabetically?

Reply:
Too many changes for this PR. Better to run `clang-format` with `hpc coding conventions` if possible at some point and also arrange them better.

Reply:
Let's do code style stuff in a separate PR all at once.